.storybook/config.js

```diff
@@ -10,10 +10,17 @@ addDecorator(story => (
   <JssProvider generateClassName={generateClassName}>{story()}</JssProvider>
 ));
 
-const req = require.context('../app/components', true, /\.stories\.js$/);
+const componentsReq = require.context(
+  '../app/components',
```

Comment:
I was trying to improve this config but I've failed to do so. I've got bitten by webpack require 🤣 

I now just copied this config to in to my branch to avoid further conflicts instead

Reply:
Sure, no problemo